qlever_dumps/props/props_text.py

```
@@ -0,0 +1,458 @@
+    # ---
+    with open(qids_file, "r", encoding="utf-8") as file:
+        qids = json.load(file)
+    # ---
+    new_data["properties"][pid]["len_of_qids"] = len(qids.get("qids", {}))
+    # ---
+    new_qids = fix_others(pid, qids.get("qids", {}))
+    # ---
+    return new_qids
+
+
+def make_section(pid, table, old_data, max_n=51):
```

Ruff (0.13.1)

Unused function argument: `max_n`

(ARG001)